Build fails on Fedora 23

```
[  1%] Building CXX object contrib/libs/stlport/CMakeFiles/contrib-libs-stlport.dir/stlport-5.1.4/src/time_facets.cpp.o
/somewhere/tomita-parser/src/contrib/libs/stlport/stlport-5.1.4/src/time_facets.cpp: In function 'char* NStlPriv::__write_formatted_time(char*, size_t, char, char, const NStlPriv::_Time_Info&, const tm*)':
/somewhere/tomita-parser/src/contrib/libs/stlport/stlport-5.1.4/src/time_facets.cpp:379:19: error: 'const struct tm' has no member named '__tm_gmtoff'
         diff = t->__tm_gmtoff;
                   ^
```

--- a/src/contrib/libs/stlport/stlport-5.1.4/src/time_facets.cpp
+++ b/src/contrib/libs/stlport/stlport-5.1.4/src/time_facets.cpp
@@ -373,7 +373,7 @@ char * _STLP_CALL __write_formatted_time(char* buf, size_t buf_size, char format
         break;
       {
         int diff;
-#    if defined (__USE_BSD) || defined (__BEOS__)
+#    if defined(__USE_MISC) || defined (__USE_BSD) || defined (__BEOS__)
         diff = t->tm_gmtoff;
 #    else
         diff = t->__tm_gmtoff;

